packages/connect-form/src/components/advanced-options-tabs/ssh-tunnel-tab/ssh-tunnel-identity.tsx

```diff
@@ -0,0 +1,124 @@
+import { TextInput, FileInput, spacing, Icon } from '@mongodb-js/compass-components';
+import { SSHConnectionOptions } from '../../../hooks/connection-options-handler';
+
+const inputFieldStyles = css({
+  width: '50%',
+  marginBottom: spacing[3],
+});
+const fileHelpStyles = css({
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'start',
+  alignItems: 'center',
+});
+
+type IdentityFormKeys = keyof SSHConnectionOptions;
+type IdentityFormErrors = {
+  [key in IdentityFormKeys]?: string;
+}
+
+function Identity({
+  sshTunnelOptions,
+  onConnectionOptionChanged,
+  errors,
+}: {
+  sshTunnelOptions: ConnectionOptions['sshTunnel'];
+  onConnectionOptionChanged: (key: IdentityFormKeys, value: string | number) => void;
+  errors?: IdentityFormErrors;
+}): React.ReactElement {
+  const formFieldChanged = (key: IdentityFormKeys, value: string | number) => {
+    onConnectionOptionChanged(key, value);
+  }
+  return (
+    <>
+      <TextInput
+        onChange={(event: ChangeEvent<HTMLInputElement>) => {
+          formFieldChanged('host', event.target.value);
+        }}
+        className={inputFieldStyles}
+        key={'host'}
+        label={'SSH Hostname'}
+        type={'text'}
+        optional={false}
+        placeholder={'SSH Hostname'}
+        value={sshTunnelOptions?.host}
+        errorMessage={errors?.host}
+        state={errors?.host ? 'error' : 'none'}
+      />
+      <TextInput
+        onChange={(event: ChangeEvent<HTMLInputElement>) => {
+          formFieldChanged('port', Number(event.target.value));
+        }}
+        className={inputFieldStyles}
+        key={'port'}
+        label={'SSH Tunnel Port'}
+        type={'number'}
+        optional={false}
+        placeholder={'SSH Tunnel Port'}
+        value={(sshTunnelOptions?.port ?? '').toString()}
+        errorMessage={errors?.port}
+        state={errors?.port ? 'error' : 'none'}
+      />
+      <TextInput
+        onChange={(event: ChangeEvent<HTMLInputElement>) => {
+          formFieldChanged('username', event.target.value);
+        }}
+        className={inputFieldStyles}
+        key={'username'}
+        label={'SSH Username'}
+        type={'text'}
+        optional={false}
+        placeholder={'SSH Username'}
+        value={sshTunnelOptions?.username}
+        errorMessage={errors?.username}
+        state={errors?.username ? 'error' : 'none'}
+      />
+      <FileInput
+        onChange={(files: string[]) => {
+          formFieldChanged('identityKeyFile', files[0]);
+        }}
+        label={'SSH Identity File'}
+        error={errors?.identityKeyFile}
+        values={
+          sshTunnelOptions?.identityKeyFile && sshTunnelOptions.identityKeyFile
+          ? [sshTunnelOptions.identityKeyFile]
+          : undefined
+        }
+        helpText={(
+          <div className={fileHelpStyles}>
+            <a href="https://mongodb.com">Learn More</a>
+            <Icon glyph="OpenNewTab" />
+          </div>
```

Comment:
There is a leafygreen link component in the [Typography](https://www.mongodb.design/component/typography/example/) with "open in tab" icon that we probably want to use here for consistency